src/utils/nameUtils.js

```diff
@@ -0,0 +1,9 @@
+export function deriveIndividualName(
+  individualData,
+  context = 'defaultName',
+  fallback = undefined,
```

Comment:
`fallback` will default to undefined automatically. It shouldn't require an explicit assignment.